### DISCUSSION TITLE
keyball39 trackball does not work

### DISCUSSION THREAD

**Discussion by dorongss** [Yowkees/keyball#474] [Q&A]
The trackball doesn't work at all.
I don't know what the hell the problem is. Please solve it.
If there's no solution
Please tell me how I can buy related parts again
![KakaoTalk_20240123_195123730](https://github.com/Yowkees/keyball/assets/157365330/0d78cae2-b142-4f14-9d4c-ead4e53d8112)
![KakaoTalk_20240123_195123730_01](https://github.com/Yowkees/keyball/assets/157365330/94fd3bea-a55d-4135-a853-cecae2b271f7)
![KakaoTalk_20240123_195123730_02](https://github.com/Yowkees/keyball/assets/157365330/cac5589a-cd41-4d58-85d7-4d4e69739a60)

**Comment by Ryo624** [comment]
Thank you for contacting us.

The soldering on the trackball board appears to be OK, so try re-soldering the　L-shaped Conthrough pin and jumper area.

Note that the place where the L-shaped Conthrough pin is inserted into the trackball board is a place where soldering is not required.
